auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -359,7 +363,9 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
      * silently.
      *
      * @param activity the calling {@link Activity}.
+     * @deprecated use {@link #delete(FragmentActivity)} instead
      */
+    @Deprecated
     public Task<Void> delete(@NonNull Activity activity) {
```

Comment:
@samtstern To address this comment and [this one](https://github.com/firebase/FirebaseUI-Android/pull/566#discussion_r98960229), I didn't add an interface or a good name because my intention for us to throw this away in FBUI `v2.0`. Do you disagree with that? If so, then we should definitely clean things up and maybe undeprecate thoses classes. What do you think?